name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/PDFParser.java

```diff
@@ -192,6 +219,12 @@ public void parse(String filename, List<Item> items, String[] lines, int lineNoS
     /* package */static class Section<T>
     {
         private boolean isOptional = false;
+        
+        
+        private boolean isGroup = false;
+        private int groupID;
+        private String groupError = null;
```

Comment:
Bisher sind die Extractors stateless in dem Sinne als das mit einer Extractor Instanz mehrere PDF Dokumente in einem Lauf importieren kann. Noch gibt es State weil der Extractor den Client aber auch den SecurityCache speichert. Das soll sich aber ändern. Gerade hat @akoch einen PR #916 gestellt, der den nächsten Schritt machen (habe ich noch nicht im Detail durchgeschaut). Darum würde ich hier jetzt ungerne neuen State einführen.

Könnte man diese Information nicht weiter oben in Transaction#parse sammeln? Sprich: wenn es sich um einen Block handelt, dann Exception nicht durchreichen, sondern sammeln und am Ende checken das genau nur ein Block erfolgreich war.